Dockerfile

```diff
@@ -19,11 +19,12 @@ RUN curl https://mise.run | sh
 
 # Install Node.js, Python, and Go.
 RUN mise install node@latest && mise use --global node@latest
-RUN mise install python@latest && mise use --global python@latest
+RUN mise install python@3.9.5 && mise use --global python@3.9.5
```

Comment:
Just curious, why not latest? (That's what we have in `mise.toml`, and ideally these would be kept in sync.)

Reply:
The Python SDK has a version file at 3.9.5 and I was getting an error when trying to install dependencies in the container before the steps. I took that piece out and but forgot about this pinning to the version.